src/NHSD.BuyingCatalogue.Documents.API/Startup.cs

```diff
@@ -56,6 +58,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
                     });
             }
 
+            app.UseHealthChecks("/health");
```

Comment:
Also in the Buying Catalogue API the health checks are split by live and dependencies. This allows a third party to determine whether or not the API is active and its dependencies are ready.